### PR TITLE
input sizing

### LINKED ISSUE #174: Fix search box style on clip details page
above the largest breakpoint, it looks like this.
image

### PR DESCRIPTION
**Pull request by mccalluc**
I think this covers all the bases, or at least suggests everything that needs to be checked. Fix #174?

### CODE REVIEW

**Comment by afred** [comment]
I'll pull this down and take a look. Stand by.

**Comment by afred** [comment]
@mccalluc please review

**Comment by mccalluc** [comment]
@afred: The input in the call-to-action when sm or xs should be wider, I think, but that could be handled separately? ... There's some other work that needs to be done here. Otherwise, fine to merge.
screen shot 2015-10-21 at 11 52 34 am

File: app/assets/stylesheets/locals/call_to_action.css.scss [hunk 1]
```diff
@@ -25,6 +25,8 @@
     }
     .well {
         width: 100%;
+        min-height: 120px;
+
```
Comment (mccalluc) [inline app/assets/stylesheets/locals/call_to_action.css.scss]: Do we need both this and the 4.5em below? They seem redundant.

File: app/assets/stylesheets/locals/call_to_action.css.scss [hunk 2]
```diff
@@ -51,8 +53,11 @@
...
     .call_to_action .well {
-        a,form {
-            height: 4.5em; // Can accomodate 3 lines of text
-        }
+
+        // min-height: 120px;
+
+        // a,form {
+        //     height: 4.5em; // Can accomodate 3 lines of text
+        // }
```
Comment (mccalluc) [inline app/assets/stylesheets/locals/call_to_action.css.scss]: Can we delete all this?

File: app/assets/stylesheets/locals/search_form.css.scss [hunk 3]
```diff
@@ -1,21 +1,69 @@
...
+      }
+    }  
+  } 
+}
+
+
+// Override search_form width in .call_to_action .well
+.call_to_action {
+  .well {
+    .search_form {
```
Comment (mccalluc) [inline app/assets/stylesheets/locals/search_form.css.scss]: style question: Do we prefer class chains like this broken out rather on a single line?

File: app/assets/stylesheets/locals/search_form.css.scss [hunk 4]
```diff
@@ -1,21 +1,69 @@
...
+
+      @media (min-width: $screen-md-min) {
+          width: 220px;
+      }
+      @media (min-width: $screen-lg-min) {
+          width: 300px;
+      }
```
Comment (mccalluc) [inline app/assets/stylesheets/locals/search_form.css.scss]: The same declaration is made on line 8... Is one or the other redundant?

File: app/assets/stylesheets/locals/search_form.css.scss (hunk 3 above)
Comment (afred) [inline app/assets/stylesheets/locals/search_form.css.scss]: @mccalluc broken out, flexible+readable imho.

File: app/assets/stylesheets/locals/call_to_action.css.scss (hunk 2 above)
Comment (afred) [inline app/assets/stylesheets/locals/call_to_action.css.scss]: yes. I'll do it.

File: app/assets/stylesheets/locals/search_form.css.scss (hunk 4 above)
Comment (afred) [inline app/assets/stylesheets/locals/search_form.css.scss]: the one specific to `.call-to-action` appears to be redundant. Will remove.

File: app/assets/stylesheets/locals/call_to_action.css.scss (hunk 1 above)
Comment (afred) [inline app/assets/stylesheets/locals/call_to_action.css.scss]: @mccalluc re: units.. i've been convinced to ditch `.em` altogether. I can't think of a good use case now that modern browsers handle resizing as well as they do. And for that matter, using anything except `px` seems to need a special reason.

**Comment by afred** [comment]
@mccalluc removed redundant/unused styles. please review

**Comment by afred** [comment]
also.. agreed there are a few more responsiveness tweaks that need to be made, but leaving for a different ticket, b/c i think it will touch more code.

**Pull request merged**